RXPatchLib/UpdateServerHandler.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RxLogger;
+
+namespace RXPatchLib
+{
+
+    /// <summary>
+    /// Contains a server entry that eventually sits inside a List array
+    /// </summary>
+    public class UpdateServerEntry
+    {
+        public Uri Uri;
+        public string Name;
+        public bool IsUsed;
+        public bool HasErrored;
+        public string WebPatchPath;
```

Comment:
Why do we need WebPatchPath here? It seems like it'd be better to have this live on RXPatcher